packages/cardano-services/test/NetworkInfo/NetworkInfoHttpService.test.ts

```diff
@@ -320,11 +321,19 @@ describe('NetworkInfoHttpService', () => {
       });
 
       it('lovelaceSupply', async () => {
+        const { maxLovelaceSupply } = await loadGenesisData(cardanoNodeConfigPath);
         const response = await provider.lovelaceSupply();
-        expect(response.total).toBeGreaterThan(0);
-        expect(response.circulating).toBeGreaterThan(0);
-        // FIXME: fix circulating supply calculation
-        // expect(response.total).toBeGreaterThan(response.circulating);
+        expect(response.circulating).toBeGreaterThan(0n);
+        expect(response.total).toBeLessThan(maxLovelaceSupply);
+      });
+
+      // FIXME: There is a db-sync issue with the values in `ada_pots` table:
+      //        https://github.com/input-output-hk/cardano-db-sync/issues/942
+      it.skip('lovelaceSupply (with better assertions)', async () => {
+        const { maxLovelaceSupply } = await loadGenesisData(cardanoNodeConfigPath);
+        const response = await provider.lovelaceSupply();
+        expect(response.total).toBeGreaterThan(response.circulating);
```

Comment:
```suggestion
        expect(response.total).toBeGreaterThan(response.circulating); // This assertion is to be at the above test case. (i.e. Add at line 327)
```

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/337/commits/d4b18688bcab0cb6b80ccf44c8f1cbc22705cca4